react/week5/github-search/src/header.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import "./index.css";
+import { BrowserRouter } from "react-router-dom";
+
+class Header extends React.Component {
+  render() {
+    return (
+      <BrowserRouter>
```

Comment:
There aren't any routes within this router. Was it an accident that you're using this router here? 

It isn't useful unless composed together with `Router`

Reply:
Yes, sorry it was the idea to use browserrouter with switch to jump into selected users detail page, then ichanged my mind and i did in the body part in index except in the header. I have deleted it now.